hipblas/PKGBUILD

```diff
@@ -1,16 +1,16 @@
 # Maintainer: Markus Näther <naether.markus@gmail.com>
 pkgname=hipblas
-_pkgver=3.3
-pkgver="$_pkgver.0"
-pkgrel=1
-pkgdesc="Next generation BLAS implementation for ROCm platform"
+pkgver=3.3.0
+pkgrel=2
+pkgdesc='ROCm BLAS marshalling library'
 arch=('x86_64')
-url="https://github.com/ROCmSoftwarePlatform/hipBLAS"
+url='https://rocmdocs.amd.com/en/latest/ROCm_Libraries/ROCm_Libraries.html#hipblas'
 license=('MIT')
-depends=('hcc' 'hip')
-makedepends=('cmake' "hcc>=$pkgver" 'python2' "comgr>=$pkgver" 'rocminfo')
-source=("https://github.com/ROCmSoftwarePlatform/hipBLAS/archive/rocm-$pkgver.tar.gz"
-        "hipblas_hsa.patch")
+depends=('rocblas' 'hip-hcc')
+makedepends=('cmake' 'hcc')
+_git='https://github.com/ROCmSoftwarePlatform/hipBLAS'
+source=("$pkgname-$pkgver.tar.gz::$_git/archive/rocm-$pkgver.tar.gz"
+        'hipblas_hsa.patch')
```

Comment:
Do you know why we have this patch? Can you make an issue on the build error without this patch so that we can track it and report to upstream if needed?

Reply:
`hipblas` doesn't compile without the patch. It seems to be an issue with HIP as it needs HSA. But I haven't figured out yet why HIP does not include HSA. The include files are at `/opt/rocm/hsa/include` with a symlink to `/opt/rocm/include`, in accordance with the ubuntu packages.

Reply:
kk, ill go through the packages some time later and track the packages we have patched.